kube/redis-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kernelci-api-redis
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kernelci-api-redis
+  template:
+    metadata:
+      labels:
+        app: kernelci-api-redis
+    spec:
+      containers:
+        - name: kernelci-api-redis
+          image: redis:6.2
+          volumeMounts:
+            - name: redis-data
+              mountPath: /data
+          # Resource limits needs to be discussed.
+          resources:
+  volumes:
+    - name: redis-data
+      hostPath:
+        path: home/docker/kernelci-api/redis/data
```

Comment:
Same comment about indentation.

Reply:
They are part of the spec already. I think you confused it with the template spec which is a different thing.

Reply:
I am also talking about `spec.template.volumes` field.
Please check the indentation of `volumes` in `api-deployment.yaml` file. That is the correct one.
It should be the same for other deployment files.

Reply:
Will fix it. It was unnoticeable as I was fixing the indent manually in VS code using ctrl + ] so probably missed this. Thanks for pointing it out.